app/src/main/java/com/keylesspalace/tusky/entity/Account.kt

```diff
@@ -23,6 +23,7 @@ data class Account(
     @SerializedName("username") val localUsername: String,
     @SerializedName("acct") val username: String,
     @SerializedName("display_name") val displayName: String?, // should never be null per Api definition, but some servers break the contract
+    @SerializedName("created_at") val createdAt: String,
```

Comment:
just use (java.util.)Date here

Reply:
Ahh ok, I wasn't sure if storing Date on the DB was ok (some people prefer to store the raw Strings/values). I'll update it alongside the MIgration. Seems like I pushed _seconds_ before you reviewed it (unfortunately).

Reply:
Done